diesel_tests/tests/bench.rs

```diff
@@ -1,5 +1,8 @@
 #![feature(custom_derive, plugin, custom_attribute, test)]
 #![plugin(diesel_codegen, dotenv_macros)]
+
+#![allow(non_snake_case)] // benchmark names contain several underscores to align numbers
```

Comment:
Can you just mention this in the commit message instead of a code comment? :smile: